src/protocolsupport/ProtocolSupport.java

```diff
@@ -65,7 +65,7 @@ public void onLoad() {
 			BlockStorageReader.init();
 			ServerPlatform.get().inject();
 			PESkin.init();
-			server = new MCPEServer(19132);
+			server = new MCPEServer(2222);
```

Comment:
@NycuRO I could but that's up to @Shevchik. For now at least you'll have to change that line before you compile Protocol Support.